packages/react/src/themes/teams/components/SplitButton/splitButtonStyles.ts

```diff
@@ -1,54 +1,52 @@
 import { ICSSInJSStyle } from '../../../types'
-import getBorderFocusStyles from '../../getBorderFocusStyles'
-import SplitButton from '../../../../components/SplitButton/SplitButton'
+import getIconFillOrOutlineStyles from '../../getIconFillOrOutlineStyles'
 
 const splitButtonStyles = {
-  button: ({ variables: v }): ICSSInJSStyle => ({
-    border: 0,
+  menuButton: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    borderTopRightRadius: '0',
+    borderBottomRightRadius: '0',
+    borderRightWidth: '0',
     padding: v.padding,
 
-    ':focus-visible': {
-      '::before': { border: 0 },
-      '::after': { border: 0 },
+    ...(p.small && {
+      height: v.smallDimension,
+      padding: v.smallPadding,
+      minWidth: v.smallMinWidth,
+      boxShadow: v.smallBoxShadow,
+    }),
+
+    ':active': {
+      animationName: '0',
```

Comment:
This probably isn't ideal but makes it so the button doesn't animate when clicked.

Reply:
`animationName: 'unset'` should be used here, and please unser the other animation* related props